cluster-api/src/main/java/io/aiven/klaw/clusterapi/controller/ClusterApiController.java

```diff
@@ -0,0 +1,252 @@
+
+    Map<String, String> resultMap = new HashMap<>();
+    String result;
+    try {
+      if (AclsNativeType.NATIVE.name().equals(clusterAclRequest.getAclNativeType())) {
+        if (AclType.PRODUCER.value.equals(clusterAclRequest.getAclType())) {
+          result = apacheKafkaAclService.updateProducerAcl(clusterAclRequest);
+        } else {
+          result = apacheKafkaAclService.updateConsumerAcl(clusterAclRequest);
+        }
+        return new ResponseEntity<>(ApiResponse.builder().result(result).build(), HttpStatus.OK);
+      } else if (AclsNativeType.AIVEN.name().equals(clusterAclRequest.getAclNativeType())) {
+        resultMap = aivenApiService.createAcls(clusterAclRequest);
+        return new ResponseEntity<>(
+            ApiResponse.builder().result(resultMap.get("result")).data(resultMap).build(),
+            HttpStatus.OK);
+      }
+    } catch (Exception e) {
+      resultMap.put("result", "failure " + e.getMessage());
+      return new ResponseEntity<>(
+          ApiResponse.builder().result("failure " + e.getMessage()).build(),
+          HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+    resultMap.put("result", "Not a valid request");
+    return new ResponseEntity<>(
+        ApiResponse.builder().result("Not a valid request").build(),
+        HttpStatus.INTERNAL_SERVER_ERROR);
+  }
+
+  @PostMapping(value = "/deleteAcls")
+  public ResponseEntity<ApiResponse> deleteAcls(
+      @RequestBody @Valid ClusterAclRequest clusterAclRequest) {
+    String result;
+    try {
+      if (AclsNativeType.NATIVE.name().equals(clusterAclRequest.getAclNativeType())) {
+
+        if (AclType.PRODUCER.value.equals(clusterAclRequest.getAclType())) {
+          result = apacheKafkaAclService.updateProducerAcl(clusterAclRequest);
+        } else {
+          result = apacheKafkaAclService.updateConsumerAcl(clusterAclRequest);
+        }
+
+        return new ResponseEntity<>(ApiResponse.builder().result(result).build(), HttpStatus.OK);
+      } else if (AclsNativeType.AIVEN.name().equals(clusterAclRequest.getAclNativeType())) {
+        result = aivenApiService.deleteAcls(clusterAclRequest);
+
+        return new ResponseEntity<>(ApiResponse.builder().result(result).build(), HttpStatus.OK);
+      }
+
+    } catch (Exception e) {
+      return handleException(e);
+    }
+    return new ResponseEntity<>(
+        ApiResponse.builder().result("Not a valid request").build(),
+        HttpStatus.INTERNAL_SERVER_ERROR);
+  }
+
+  @PostMapping(value = "/postSchema")
+  public ResponseEntity<ApiResponse> postSchema(
+      @RequestBody @Valid ClusterSchemaRequest clusterSchemaRequest) {
+    try {
+      return new ResponseEntity<>(
+          schemaService.registerSchema(clusterSchemaRequest), HttpStatus.OK);
+    } catch (Exception e) {
+      return handleException(e);
+    }
+  }
+
+  private static ResponseEntity<ApiResponse> handleException(Exception e) {
+    log.error(e.getMessage());
```

Comment:
```suggestion
    log.error("Exception:", e);
```
This logs the whole stacktrace  rather than just a message